site/_apps/zuix-hn.txt

```diff
@@ -14,11 +14,11 @@ hosting: 'Firebase'
 authors: 
     - name: 'Gene'
 lighthouse: '91/100'
-interactive-em: '5.3s'
-interactive-faster-3g: '5.0s'
-lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170608_R9_da7e6d2250380cec54c9555a0b59b69a&run=3
-wpt-em-link: https://www.webpagetest.org/result/170608_WN_a3acc96da84088832cd85ea6be37da1a/
-wpt-faster-3g-link: https://www.webpagetest.org/result/170608_R9_da7e6d2250380cec54c9555a0b59b69a/
+interactive-em: '3.2s'
+interactive-faster-3g: '2.2s'
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170611_5G_d360ab4227e495c34cfb8e6ea58b37e4&run=3
+wpt-em-link: https://www.webpagetest.org/result/170611_QG_beecb3a95d8b5e66ed6a33522b1a20a7/
```

Comment:
@housseindjirdeh @kristoferbaxter There's something a little strange going on in the very latest trace over at https://www.webpagetest.org/video/compare.php?tests=170611_QG_beecb3a95d8b5e66ed6a33522b1a20a7-r:1-c:0.

I see:
- main-thread activity settling down initially after 3.7s (consistently after 7.5/8s)
- earliest the app displays interactive UI is about 6s

This isn't OP's fault at all as Lighthouse reports different numbers here. Based on what I've seen upstream, we might want to wait until latest Lighthouse (2.1.0) gets deployed to WPT and re-run the numbers then before landing.

Does that sound reasonable?